pkgs/development/python-modules/hsh/default.nix

```diff
@@ -0,0 +1,38 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, commandlines
+, unittestCheckHook
+, pexpect
+, naked
+, nix-update-script
+}:
+
+buildPythonPackage rec {
+  pname = "hsh";
+
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "chrissimpkins";
+    repo = "hsh";
+    rev = "v${version}";
+    hash = "sha256-bAAytoidFHH2dSXqN9aqBd2H4p/rwTWXIZa1t5Djdz0=";
+  };
+
+  propagatedBuildInputs = [ commandlines ];
+
+  nativeCheckInputs = [ unittestCheckHook pexpect naked ];
+
+  preCheck = "cd tests";
+
+  passthru.updateScript = nix-update-script {};
+
```

Comment:
```suggestion
```
There is no need to pass a simple `nix-update-script` since the python package has a default updateScript.
https://github.com/NixOS/nixpkgs/blob/344705ea5d1a24d909ed9b0f6fcfd6b4d10bb356/pkgs/development/interpreters/python/mk-python-derivation.nix#L300-L302

Reply:
Yes, but in my tests it only works if src is built with `fetchPypi`. `fetchFromGitHub` doesn't work.

Reply:
It will not give errors if ausent but  the new version will not be found.